### DISCUSSION TITLE
Upload to the board directly from the browser, and lgt based web-oscilloscope!

### DISCUSSION THREAD

**Discussion by dbuezas** [dbuezas/lgt8fx#85] [Show and tell]
# This is not an issue, but I have no other means to get feedback on these lgt8f328p related projects:

Check out the lgt-oscilloscope at 
* direct access [chrome only]: https://dbuezas.github.io/arduino-web-oscilloscope/
* code: https://github.com/dbuezas/arduino-web-oscilloscope/
* you can flash the board from chrome itself!
* I'll eventually make an stm32 version, which will extend the bandwidth from 200kHz to ~1MHz. 
<img width="1348" alt="screenshot" src="https://user-images.githubusercontent.com/777196/97110199-25ed7c80-16d8-11eb-8562-a60050e0afe0.png">

---

And if you also want to "upload code from chrome", check out this one
* example [chrome only]: https://dbuezas.github.io/arduino-web-uploader/
* code & usage: https://github.com/dbuezas/arduino-web-uploader
I know some of you are good at C++ but don't care about javascript, so I made a "plugin" of sorts so you can add an upload button to your html-only page with no code.
![Screen Shot 2020-10-25 at 15 39 17](https://user-images.githubusercontent.com/777196/97110237-4c131c80-16d8-11eb-9ec6-0ef367eb0740.png)

--

I'll appreciate feedback in the respective repos (and sorry for spamming this one)

**Comment by Pittyalex** [comment]
It's super.
